results/debug-oracle.json

```diff
@@ -971,18 +974,18 @@
         },
         "37615223": {
           "banned_phrases": [
-            "third",
-            "the Amex Stadium"
+            "the Amex Stadium",
+            "third"
           ],
           "summary": "Sam Baldock scored the only goal of the game as Brighton beat Wolves to move up to fourth in the Championship.",
           "generation_metadata": {
-            "score": -0.5369529128074646,
+            "score": -0.53695148229599,
             "dropped_seqs": [
               "</s>Sam Baldock scored the only goal of the game as Brighton beat Wolves at the Amex Stadium",
               "</s>Sam Baldock scored the only goal of the game as Brighton beat Wolves to move up to third",
               "</s>Sam Baldock's first-half header was enough to give Brighton victory over Wolves at the Amex Stadium"
             ],
-            "n_words_checked": 96
+            "n_words_checked": 81
```

Comment:
why would we expect any diff here? Was thinking the NER in https://github.com/danton-nlp/factual-beam-search/pull/41 - but that's a separate Git history.

Reply:
I think the issue is that we didn't re-generate `debug-oracle.json` consistently at the end of the deadline push - we were testing & debugging directly with 
the final test-extrinsic set. 

This means that this diff doesn't only capture changes introduced in this PR, but let me rerun debug on main to verify

Reply:
Gotcha, yeah, the debug dset doesn't serve any important purpose anymore either, so all good regardless methinks

Reply:
Reran on main & rebased. These stats dont change anymore